packages/shared/src/components/drawers/StorybookDrawerTrigger.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement, useState } from 'react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { Drawer, DrawerProps } from './Drawer';
+
+export function StorybookDrawerTrigger({
```

Comment:
Can't we add this directly to storybook package?
Feels a bit misplaced now.

Reply:
It is missing quite a lot of packages, one for react-query, if we want, we can install those deps as well.

Reply:
Hmm let's see what @nensidosari thinks.
Don't want to overcomplicate, might just look weird to newcomers

Reply:
Yeah, agree with that.

Reply:
Also don't want this here. I'm open to suggestions on how we can tackle this. Maybe the right way is by installing the deps? Not sure yet at the moment 😅

Reply:
Why do we need `QueryClientProvider` in here?

Reply:
The Storybook is crashing without it. We have some internal hooks that utilize the query client.

Reply:
I think this is the reason: https://github.com/dailydotdev/apps/blob/MI-124-drawer-component/packages/shared/src/components/tooltips/Portal.tsx#L25

Do we. need to use the `RootPortal`? Maybe that could be a quick fix :)

Reply:
Arguably, should be possible not to use it. There could be some implementations in the future where the Drawer might not work without it, but we can probably tackle that when it comes.

Reply:
Yep. Have decided to remove the Portal and bring it back once necessary.